gallery/tutorials/lecture_feature_demo.py

```diff
@@ -388,25 +349,17 @@ def noise_function(x, y):
     max_rows=1024,
 )
 
-# Data resides on Tiger Cluster
-# Please make sure you are using a compute node once you've installed ASPIRE, not the head node...
-# src =  RelionSource(
-#    "/tigress/gbwright/data/cryo-em/CryoEMdata/empiar10028/shiny_2sets.star", data_folder="", pixel_size=5.0, max_rows=100
-# )
 src.downsample(img_size)
 
 src.images[:10].show()
 
-noise_estimator = WhiteNoiseEstimator(src)
-src.whiten(noise_estimator.filter)
-
 orient_est = CLSyncVoting(src, n_theta=36)
 orient_est.estimate_rotations()
 rots_est = orient_est.rotations
 
 # %%
 # We can see that the code can easily run with experimental data by subsituting the ``Source`` class.
-# However, we have hit the practical limitation that requires class averaging of images....
+# However, we have hit the point where we need denoising algorithms to perform orientation estimation.
```

Comment:
Currently the logger info messages containing the spectral info and mse of rotations do not show. I believe this comment is related to those values changing when using experimental data. Should we try to have those values displayed in the tutorial?

Reply:
The eigenvalues appear to be in the box directly above for me?  Did you mean something else?  This is probably enough information without MSE.  If the MSE is good or not when the algorithm reports its known unsuccessful is just happenstance...

<img width="777" alt="Screen Shot 2023-02-20 at 1 23 31 PM" src="https://user-images.githubusercontent.com/47759732/220177658-7df09756-c256-4149-8afd-35392fdfe145.png">

Reply:
hmm, interesting. For some reason this log does not appear for me when I build the docs. Just checked the actual docs and here's what it shows:
<img width="742" alt="Screen Shot 2023-02-21 at 8 28 54 AM" src="https://user-images.githubusercontent.com/66811911/220357608-ce0d3c5c-8db6-4dab-84a0-14fa73629f3d.png">

Reply:
Sure, but that screen shot is not from this PR...

If it is happening locally, that is really strange.  Can you post this area your actual doc? If it is the same, try `make distclean` then remake ? Also please report back with the top of your `git log` your `pip freeze`. 

Maybe there is something different between our setups... Sorry for the extra work, but I'd like to figure this out while the PR is open.

Reply:
Thanks, that was after the remake? I'm trying to rule out that we are actually looking at exactly the same code...

Reply:
lol, three different outputs.  Galleries are fun.

Reply:
Top of `git log`:
```
commit d63adf12e5257fdcd79bdbd4672ce6bde33fdfcb (HEAD -> lecture_demo_updates, origin/lecture_demo_updates)
Author: Garrett Wright <garrettwrong@gmail.com>
Date:   Thu Feb 16 10:28:51 2023 -0500

    Remove ugly plots

commit 42d8b97570553d24eb012183c5d7ba83d331068d
Author: Garrett Wright <garrettwrong@gmail.com>
Date:   Thu Feb 16 10:14:53 2023 -0500

    Remove tasks/homework prompts

commit 8720ce2ea26d3990735677f1c72bfd26634ff31e
Merge: 555a411e 4c20d7f6
Author: Garrett Wright <47759732+garrettwrong@users.noreply.github.com>
Date:   Fri Feb 10 15:09:20 2023 -0500

    Merge pull request #853 from ComputationalCryoEM/neg_ind
    
    Negative indexed slice bug

```

`pip freeze`:
```
aiosignal==1.3.1
alabaster==0.7.13
anyio==3.6.2
appnope==0.1.3
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
-e git+ssh://git@github.com/ComputationalCryoEM/ASPIRE-Python.git@d63adf12e5257fdcd79bdbd4672ce6bde33fdfcb#egg=aspire
asttokens==2.2.1
attrs==22.2.0
Babel==2.11.0
backcall==0.2.0
beautifulsoup4==4.11.2
black==23.1.0
bleach==6.0.0
build==0.10.0
bump2version==1.0.1
bumpversion==0.6.0
cachetools==5.3.0
certifi @ file:///private/var/folders/sy/f16zz6x50xz3113nwtb9bvq00000gp/T/abs_477u68wvzm/croot/certifi_1671487773341/work/certifi
cffi==1.15.1
chardet==5.1.0
charset-normalizer==3.0.1
check-manifest==0.49
click==8.1.3
colorama==0.4.6
comm==0.1.2
confuse==2.0.0
contourpy==1.0.7
coverage==7.1.0
cycler==0.11.0
debugpy==1.6.6
decorator==5.1.1
defusedxml==0.7.1
distlib==0.3.6
docutils==0.18.1
exceptiongroup==1.1.0
execnet==1.9.0
executing==1.2.0
fastjsonschema==2.16.2
filelock==3.9.0
finufft==2.1.0
flake8==6.0.0
fonttools==4.38.0
fqdn==1.5.1
frozenlist==1.3.3
gemmi==0.5.8
grpcio==1.51.1
idna==3.4
imageio==2.25.1
imagesize==1.4.1
importlib-metadata==6.0.0
importlib-resources==5.10.2
iniconfig==2.0.0
ipykernel==6.21.2
ipython==8.10.0
ipython-genutils==0.2.0
ipywidgets==8.0.4
isoduration==20.11.0
isort==5.12.0
jaraco.classes==3.2.3
jedi==0.18.2
Jinja2==3.1.2
joblib==1.2.0
jsonpointer==2.3
jsonschema==4.17.3
jupyter==1.0.0
jupyter-console==6.5.1
jupyter-events==0.6.3
jupyter_client==8.0.3
jupyter_core==5.2.0
jupyter_server==2.3.0
jupyter_server_terminals==0.4.4
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.5
keyring==23.13.1
kiwisolver==1.4.4
latexcodec==2.0.1
markdown-it-py==2.1.0
MarkupSafe==2.1.2
matplotlib==3.7.0
matplotlib-inline==0.1.6
mccabe==0.7.0
mdurl==0.1.2
mistune==2.0.5
more-itertools==9.0.0
mrcfile==1.4.3
msgpack==1.0.4
mypy-extensions==1.0.0
nbclassic==0.5.1
nbclient==0.7.2
nbconvert==7.2.9
nbformat==5.7.3
nest-asyncio==1.5.6
networkx==3.0
notebook==6.5.2
notebook_shim==0.2.2
numpy==1.21.5
packaging==23.0
pandas==1.3.5
pandocfilters==1.5.0
parameterized==0.8.1
parso==0.8.3
pathspec==0.11.0
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.4.0
pkginfo==1.9.6
pkgutil_resolve_name==1.3.10
platformdirs==3.0.0
pluggy==1.0.0
prometheus-client==0.16.0
prompt-toolkit==3.0.36
protobuf==4.21.12
psutil==5.9.4
ptyprocess==0.7.0
pure-eval==0.2.2
pybtex==0.24.0
pybtex-docutils==1.0.2
pycodestyle==2.10.0
pycparser==2.21
pydocstyle==6.3.0
pyFFTW==0.13.1
pyflakes==3.0.1
Pygments==2.14.0
pyparsing==3.0.9
pyproject_api==1.5.0
pyproject_hooks==1.0.0
pyrsistent==0.19.3
pytest==7.2.1
pytest-cov==4.0.0
pytest-random-order==1.1.0
pytest-xdist==3.2.0
python-dateutil==2.8.2
python-json-logger==2.0.6
pytz==2022.7.1
PyWavelets==1.4.1
PyYAML==6.0
pyzmq==25.0.0
qtconsole==5.4.0
QtPy==2.3.0
ray==2.2.0
readme-renderer==37.3
requests==2.28.2
requests-toolbelt==0.10.1
rfc3339-validator==0.1.4
rfc3986==2.0.0
rfc3986-validator==0.1.1
rich==13.3.1
scikit-image==0.19.3
scikit-learn==1.2.1
scipy==1.7.3
Send2Trash==1.8.0
six==1.16.0
snakeviz==2.1.1
sniffio==1.3.0
snowballstemmer==2.2.0
soupsieve==2.4
Sphinx==6.1.3
sphinx-gallery==0.11.1
sphinx-rtd-theme==1.2.0
sphinxcontrib-applehelp==1.0.4
sphinxcontrib-bibtex==2.5.0
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.1
sphinxcontrib-jquery==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
stack-data==0.6.2
terminado==0.17.1
threadpoolctl==3.1.0
tifffile==2023.2.3
tinycss2==1.2.1
tomli==2.0.1
tornado==6.2
tox==4.4.5
tqdm==4.64.1
traitlets==5.9.0
twine==4.0.2
typing_extensions==4.5.0
uri-template==1.2.0
urllib3==1.26.14
virtualenv==20.19.0
wcwidth==0.2.6
webcolors==1.12
webencodings==0.5.1
websocket-client==1.5.1
widgetsnbextension==4.0.5
zipp==3.13.0
```

I'll update the branch, `make distclean`, rebuild and get back to you.

Reply:
Still no log 🤷‍♂️ 
<img width="610" alt="Screen Shot 2023-02-22 at 3 57 41 PM" src="https://user-images.githubusercontent.com/66811911/220757359-d1aa88e6-6e14-42ab-9dd8-6593a1b02c33.png">

Reply:
Okay, so this was after we updated your code.  Hrmm, the mystery continues. I'll have to look at the pip env tomorrow to see if anything stands out.

It's hopefully trivial to change this, I'm just lost on why it is building differently between you, docs, and myself.

Reply:
I'm making on `caf` now in a fresh environment.  Let's see whether that will generate the logs...

Reply:
Caf also did not capture! Argg.. 

Okay well, I forced the second block of expressions to be its own cell, and then was generated on both caf and my mac.  Can you try the latest push?